near-contract-standards/src/non_fungible_token/events.rs

```diff
@@ -0,0 +1,149 @@
+use std::fmt;
+
+use near_sdk::serde::{Deserialize, Serialize};
+
+//enum that represents the data type of the EventLogJson.
+//the enum can either be an NftMintLog or an NftTransferLog
+#[derive(Serialize, Deserialize, Debug)]
+/*
+    outputs JSON in the following order: <tag>: <field>, <content>: <field_data>
+    In this case an example would be: "event": "NftMint", "data": <NftMintLog Vector Data>
+*/
+#[serde(tag = "event", content = "data")] 
+//turns all the fields into snake case (NftMint --> nft_mint)
+#[serde(rename_all = "snake_case")] 
+#[serde(crate = "near_sdk::serde")]
+//makes it so that adding new fields isn't a breaking change
+#[non_exhaustive]
+pub enum EventLogVariant {
+    NftMint(Vec<NftMintLog>),
+    NftTransfer(Vec<NftTransferLog>),
+}
+
+// Interface to capture data
+// about an event
+// Arguments
+// * `standard`: name of standard e.g. nep171
+// * `version`: e.g. 1.0.0
+// * `event`: string
+// * `data`: associate event data
+#[derive(Serialize, Deserialize, Debug)]
+#[serde(crate = "near_sdk::serde")]
+pub struct EventLog {
+    pub standard: String,
+    pub version: String,
+
+    //don't have "event": {<EventLogVariant>} in the JSON, just have the contents {<EventLogVariant>}
```

Comment:
```suggestion
    // `flatten` to not have "event": {<EventLogVariant>} in the JSON, just have the contents of {<EventLogVariant>}.
```

I don't think you need to explain the serde format in comments, but here is a suggestion if you do want to keep